src/utils/todo/action-functions.ts

```diff
@@ -103,6 +103,13 @@ export const doResumeFiling = (item: TodoItemI): void => {
       params = { 'correction-id': item.filingId.toString() }
       break
 
+    case FilingTypes.COURT_ORDER:
+      // navigate to Consent Continuation Out page of Filings UI
+      navigateFn = goToFilingsUI
+      path = `/${currentBusinessIdentifier}/court-order`
+      params = { filingId: item.filingId.toString() }
+      break
```

Comment:
I think you don't need this, because there no resuming a draft court order filing, right?